incubator/hnc/docs/user-guide/quickstart.md

```diff
@@ -654,3 +655,53 @@ the prefix "Crit," for "Critical") indicates that HNC is no longer updating the
 objects in this namespace. Instead, it's waiting for an admin to come fix the
 problems before it resumes creating or deleting objects. Conditions like these
 also show up in the HNC logs, and in its [metrics](how-to.md#admin-metrics).
+
+<a name="exceptions"/>
+
+## Keeping objects out of certain namespaces
+
+_Demonstrates: exceptions._
+
+Now let’s say your acme-org has a secret that you originally wanted to share with all the teams. We created this secret as follows:
+
+```bash
+kubectl -n acme-org create secret my-secret org-secret --from-literal=password=iamacme
+```
+
+You’ll see that my-secret is propagated to both team-a and team-b
```

Comment:
Backticks on `my-secret`, `team-a`, etc. See the rest of the quickstart for guidance. Basically any name that K8s interprets (e.g. resource type, object name, namespace name) should be backticked.

Also, end this sentence with a full colon: "to both `team-a` and `team-b`:"